18_regexp_lab/03.py

```diff
@@ -1,15 +1,9 @@
-"""
-Write a python program that takes a CSV file
-reads it line by line and prints each line
-with first and second columns reversed.
-
-Sample input:
-    Shana,Sargent,shanasargent@isoswitch.com
-    Witt,Hampton,witthampton@zaphire.com
-    Morgan,Grant,morgangrant@lotron.com
-
-Sample output:
-    Sargent,Shana,shanasargent@isoswitch.com
-    Hampton,Witt,witthampton@zaphire.com
-    Grant,Morgan,morgangrant@lotron.com
-"""
+import re
+import sys
+
+filename = sys.argv[1]
+
+with open (filename, 'r') as f:
+    for line in f:
+        sys.stdout.write(re.sub(r'(\w+),(\w+),(.*)', lambda m: m.group(2)+ ',' + m.group(1)+ ',' + m.group(3), line))
+        

```

Comment:
Good Work! As a bonus read about CSV module and try to solve the same exercise with that module:
https://docs.python.org/2/library/csv.html